src/app/controllers/locations/create.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+flnd.locationCreate = {
+    configureSubmit: function($scope, config, messenger, authService, $uibModalInstance) {
+        return function() {
```

Comment:
There are probably better patterns for this...I didn't want the primary Angular controller function to be very large. I prefer for the controller to coordinate action of other functions, rather than have significant business logic in it. Therefore I created functions in the `flnd` namespace. These are essentially global static functions, which I am not altogether happy about.